lib/model-interfaces/langchain/functions/request-handler/adapters/bedrock/mistral.py

```diff
@@ -27,6 +27,7 @@ def get_llm(self, model_kwargs={}):
             model_kwargs=params,
             streaming=model_kwargs.get("streaming", False),
             callbacks=[self.callback_handler],
+            **extra
```

Comment:
I assumed it was a miss from https://github.com/aws-samples/aws-genai-llm-chatbot/commit/817f22b5644ad803291f972bf513cb6a99412d23

extra was set but not used.